wd_helper.au3

```diff
@@ -1610,6 +1610,78 @@ Func _WD_GetBrowserPath($sBrowser)
 	Return SetError(__WD_Error($sFuncName, $iErr, Default, $iExt), $iExt, $sPath)
 EndFunc   ;==>_WD_GetBrowserPath
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_GetBrowserPID
```

Comment:
should be added to wd_demo.au3 before merging

Reply:
Please take a look on: [wd_demo.au3 + _WD_GetBrowserPID()](https://github.com/Danp2/au3WebDriver/pull/324/commits/fe55389fc5cb4b79faf97e02963b3414d38ee32c)

Reply:
Two issues --

1) You misspelled "webdriver" 🙄 
2) This will always fail because the browser doesn't get launched until later in the code 

Guessing that you didn't test this code before submitting.😦

Reply:
> 1. You misspelled "webdriver" 🙄

"WedDriver" :(

> 2\. This will always fail because the browser doesn't get launched until later in the code

Thanks. [fix _WD_GetBrowserPID usage](https://github.com/Danp2/au3WebDriver/pull/324/commits/016b172e3ddfd0204e2d4b078ed24154e878a693)